### PR TITLE
Add post about Cargo team changes

### PR DESCRIPTION
**Pull request by ehuss**
This adds an internals post about some changes in the Cargo team.

cc @rust-lang/cargo

### CODE REVIEW

File: posts/inside-rust/2022-03-25-cargo-team-changes.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+Ed has been a champion on many fronts. He has done tremendous work on
+[toml_edit](https://crates.io/crates/toml_edit) to push Cargo towards getting
+`cargo add` [merged in cargo
+proper](https://github.com/rust-lang/cargo/pull/10472). He has brought
+[clap](https://crates.io/crates/clap) to the momentous 3.0 release and
+continues to push on CLI improvements, more advanced testing tools, and much
+more!
+
+At the same time one of the pillars of our team is stepping down. <del>Alex is
+a programming robot sent back in time from the future to make sure that Rust
```
Comment (calebcartwright) [inline posts/inside-rust/2022-03-25-cargo-team-changes.md:28]: :rofl:

**Comment by Mark-Simulacrum** [comment]
@ehuss I want to check in that we have sign-off here from @alexcrichton and @epage as they're prominently mentioned in the blog post. Did that happen at a Cargo meeting or elsewhere, either on direct text or on general idea? Want to avoid surprises.

I also think it's worth considering whether there's some slightly more positive tone we might be able to adopt towards the end -- obviously, presuming it's not misleading. Maybe we can say something about looking forward to the new team members helping to expand the review capacity as they ramp up on the full breadth of Cargo?

**Review by epage — APPROVED** [review]
I'm good with my part

**Review by alexcrichton — APPROVED** [review]
Indeed looks good to me!

**Comment by joshtriplett** [comment]
@Mark-Simulacrum I think the team is comfortable with this not coming across as positive. This is *intended* to be concerning. I *am* hopeful that @epage and @weihanglo will help us on this front, but I don't expect that to put us in a position where we have the capacity for new features anytime soon.

**Comment by Mark-Simulacrum** [comment]
Ok! In part I bring it up as it might be a good opportunity to try to identify how we can change that (e.g., what can folks who want to help do). It's quite possible the answer is nothing yet, though; I'm not sure whether it make sense to make that more explicit than just asking for patience. (The reverse effect of people contacting in a storm asking what they can do may also be good to mitigate with slightly more explicit language).

In any case, happy to merge this when ready - I think the date will need to get bumped. Please ping on Zulip so it doesn't get lost in email.

File: posts/inside-rust/2022-03-25-cargo-team-changes.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+capacity to accept at this time. Reviews for PRs will be taking significantly
+longer than before. For now, Cargo will be having a freeze on any new features
+or major changes. We will still be accepting bug fixes and work on existing
+projects under active development. As capacity becomes more available, new
+features may be accepted after being approved by the Cargo Team.
+
+Cargo is a large project with many moving pieces and different use cases. The
+fact that it works reliably and that it is intuitive has been a significant
+multiplier for Rust's success. But it also means that reviewing changes needs
+to be done very carefully. It is easy for changes to on one hand break on some
```
Comment (nikomatsakis) [inline posts/inside-rust/2022-03-25-cargo-team-changes.md:50]:
```suggestion
to be done very carefully. It is easy for changes to break some
```

**Review by nikomatsakis — DISMISSED** [review]
Minor wording nit.

**Review by Mark-Simulacrum — APPROVED** [review]

**Pull request merged**